coverage/src/BamViewChart.js

```diff
@@ -100,7 +100,7 @@ function createBamView(bamHeader, data, element, broker) {
 
                     // Dispatch custom event for resetting the region inputs
                     const regionsInput = {
-                        chromosome: '',
+                        rname: '',
                         start: '',
```

Comment:
Start and end should be numbers. You can set them to null if you want to detect reset values.